test/test_autograd.py

```diff
@@ -3274,21 +3274,34 @@ class dont_convert(tuple):
     ('masked_select', (), (torch.tensor(1, dtype=torch.uint8),), 'scalar'),
     ('masked_select', (M, M), (torch.tensor(1, dtype=torch.uint8),), 'scalar_broadcast_rhs'),
     ('masked_select', (), (mask_not_all_zeros((M, M)),), 'scalar_broadcast_lhs'),
+    ('masked_fill_', (M, M), (torch.ByteTensor(M, M).bernoulli_(), 10)),
     ('masked_fill', (M, M), (torch.ByteTensor(M, M).bernoulli_(), 10)),
+    ('masked_fill_', (M, M), (torch.ByteTensor(M, M).bernoulli_(), torch.tensor(10)), 'tensor'),
     ('masked_fill', (M, M), (torch.ByteTensor(M, M).bernoulli_(), torch.tensor(10)), 'tensor'),
-    # no lhs or all broadcast on masked_fill or masked_scatter because it's always inplace
+    # The outplace version broadcast to lhs, the inplace version only broadcast to rhs
+    ('masked_fill', (M,), (torch.ByteTensor(M, M).bernoulli_(), 10), 'broadcast_lhs'),
+    ('masked_fill_', (M, M), (torch.ByteTensor(M,).bernoulli_(), 10), 'broadcast_rhs'),
```

Comment:
Isn't this already tested in the case immediately below?

Reply:
Hmm, yes, let me remove it.